source/guides/tooling/typescript-support.md

```diff
@@ -40,7 +41,10 @@ The `"types"` will tell the TypeScript compiler to only include type definitions
 
 ## Types for custom commands
 
-When adding custom commands to the `cy` object, you can add their types to avoid TypeScript errors. You can find an example in the {% url "cypress-example-todomvc custom commands" https://github.com/cypress-io/cypress-example-todomvc#custom-commands %} repo.
+When adding custom commands to the `cy` object, you can add their types to avoid TypeScript errors. You can find the simplest implementation of Cypress and typescript in this {% url "repo example here" https://github.com/omerose/cypress-support %}.
```

Comment:
Error from our text linter. You can run `npm run md-lint` in the future to see these errors before commiting.
>/root/cypress-documentation/source/guides/tooling/typescript-support.md
  44:156  ✓ error  Incorrect usage of the term: “typescript”, use “TypeScript” instead  terminology

Reply:
Made the change